cli/mender-artifact/modify.go

```diff
@@ -62,6 +67,7 @@ func modifyArtifact(c *cli.Context) (err error) {
 		}
 	}()
 
+	image.dirtyImage()
```

Comment:
I decided not to be too surgical with `modify`, as the odds are every command will need the `Artifact` to be repacked.